mail/postfix/src/opnsense/service/templates/OPNsense/Postfix/main.cf

```diff
@@ -27,8 +27,8 @@ readme_directory = no
 inet_protocols = all
 meta_directory = /usr/local/libexec/postfix
 shlib_directory = /usr/local/lib/postfix
-relay_domains = hash:/usr/local/etc/postfix/transport
-transport_maps = hash:/usr/local/etc/postfix/transport
+relay_domains = btree:/usr/local/etc/postfix/transport
```

Comment:
Can we agree to stay with hash please? There's no real need for btree and I never had a problem with hash for the last 15 years ...

Reply:
they are identical with the exception that btree is faster (in large datasets)

Reply:
http://www.postfix.org/DATABASE_README.html#types

Reply:
I'm running mail relays serving 5 msg / second with hash .. 
Btree is only more perfmant when sorted (which is not ATM). 
And if performance is a must we have to use cdb which is way more performant than hash or btree.

Reply:
Sure it is sorted. That happens when it will be brought to binary form because a tree must be sorted.